src/lib/Components/Artist/ArtistShows/__tests__/ArtistShow-tests.tsx

```diff
@@ -46,4 +46,30 @@ it("renders properly", () => {
     )
     .toJSON()
   expect(show).toMatchSnapshot()
+
+  const showPropsNullKind = {
+    href: "artsy.net/show",
+    cover_image: {
+      url: "artsy.net/image-url",
+    },
+    name: "Expansive Exhibition",
+    exhibition_period: "Jan 1 - March 1",
+    status_update: "Closing in 2 days",
+    status: "running",
+    partner: {
+      name: "Gallery",
+    },
+    location: {
+      city: "Berlin",
+    },
+  }
+
+  const showNullKind = renderer
+    .create(
+      <Theme>
+        <ArtistShow show={showPropsNullKind as any} styles={showStyles} />
+      </Theme>
+    )
+    .toJSON()
+  expect(showNullKind).toMatchSnapshot()
```

Comment:
Nice test, thanks!

I think we should put this in its own `it` block with its own name, something like "it renders with a missing show kind" or something. I think there's also an opportunity to remove some duplicate code in the props we inject in.

If we move `showProps` up to the top-level in the test file, then we can use it to build out our `showPropsNullKind` props. Something like this:

```ts
const showPropsNullKind = {
  ... showProps, // this "spreads" out the showProps object so that showPropsNullKind has all the keys and values that showProps has
  kind: null // in the event that a key is repeated, the last one in the object "wins", so this effectively overwrites the `kind: "solo"` from showProps
}
```

The spread operator is one of my favourite language features from ES6 – [here's a blog post with more info](https://dev.to/torianne02/breaking-down-es6-spread-operator-k1m). Let me know what I can clarify!

Reply:
👍 Good suggestions, let me know if this works!